### PR TITLE
use actual / on BigDecimal for constant divisions

### PR DESCRIPTION
**Pull request by avi-stripe**
This adds a test for #159 as well as a simple fix.

### CODE REVIEW

**Comment by mio-stripe** [comment]
LGTM

File: rainier-core/src/main/scala/com/stripe/rainier/compute/RealOps.scala [hunk 1]
```diff
@@ -90,6 +90,12 @@ private[compute] object RealOps {
         LogLineOps.multiply(LogLine(nc1), LogLine(nc2))
     }
 
+  def divide(left: Real, right: Real): Real =
+    (left, right) match {
+      case (Constant(x), Constant(y)) => Real(x / y)
```
Comment (mio-stripe) [inline rainier-core/src/main/scala/com/stripe/rainier/compute/RealOps.scala]: Actually, in the repl, `1.0 / 0.0` gives `Infinity` whereas `BigDecimal(1.0) / BigDecimal(0.0)` throws an exception. We should guard against this.

Comment (mio-stripe) [reply rainier-core/src/main/scala/com/stripe/rainier/compute/RealOps.scala]: Yep, `Real(1.0) / Real(0.0)` throws an exception.

Comment (avi-stripe) [reply rainier-core/src/main/scala/com/stripe/rainier/compute/RealOps.scala]: Addressed in the most recent commit, can you take another look @mio-stripe ?

**Comment by mio-stripe** [comment]
LGTM

**Pull request merged**